### ISSUE TITLE
Investigate using flatlist/sections instead of scrollview for channels

### ISSUE THREAD

**Issue by AdamRivkin** [uchicagotechteam/Cook-County-Mobile-App#16]
Might help with performance on large channels where we'll need to grab a lot of thumbnails from youtube and won't want to do it all at once

**Comment by dallonasnes** [comment]
the thumbnails will scroll horizontally, right? so on ChildPage we're going to stick with a single ScrollView and nest a few (either Scrollview or Flatlist) within the outer scroll view to look through thumbnails... is that correct?

**Comment by AdamRivkin** [comment]
Yeah, that sounds right. Atm I've got a ChannelCollection component which holds an array of RainbowChannels which each hold an array of RainbowThumbnails (on the theater_base_page branch. On the regular base page branch it's RainbowVideos at the lowest level instead of thumbnails). Currently the thumbnails are in a scrollview but a flatlist might be more performant for playlists with 100+ videos where we don't want to fetch all that info when the page loads? (Maybe not though idk)
